beetsplug/replaygain.py

```diff
@@ -1381,17 +1433,30 @@ def commands(self):
         def func(lib, opts, args):
             write = ui.should_write(opts.write)
             force = opts.force
+            jobs = opts.jobs
+
+            if self.backend_instance.do_parallel and jobs > 0:
+                from multiprocessing.pool import ThreadPool
+                self.pool = ThreadPool(jobs)
 
             if opts.album:
                 for album in lib.albums(ui.decargs(args)):
                     self.handle_album(album, write, force)
-
             else:
                 for item in lib.items(ui.decargs(args)):
                     self.handle_track(item, write, force)
 
+            if hasattr(self, 'pool'):
+                self.pool.close()
+                self.pool.join()
+                self.pool.terminate()
+
         cmd = ui.Subcommand('replaygain', help=u'analyze for ReplayGain')
         cmd.parser.add_album_option()
+        cmd.parser.add_option(
+            "-j", "--jobs", dest="jobs", type=int, default=0,
+            help=u"worker pool size"
```

Comment:
Perhaps a more helpful description:
```suggestion
            help=u"number of threads for parallel analysis"
```

Reply:
Ok.